app/views/users/new.erb

```diff
@@ -0,0 +1,8 @@
+<h1>User Sign up!</h1>
+
+<form class="user-form" action="/users/new" method="post">
+  <input type="text" name="user[first_name]" placeholder="Enter your first name">
+  <input type="text" name="user[last_name]" placeholder="Enter your last name">
+  <br>
+  <input type="submit" value="Sign Up">
```

Comment:
Should this be "Create" instead of "Sign Up"?

Reply:
Yes, you're right! Thank you:)